ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-stable.yaml

```diff
@@ -325,6 +325,17 @@ tests:
     - ref: rosa-cluster-upgrade-hcp
     - chain: openshift-e2e-test-hypershift-qe
     workflow: rosa-lifecycle-advanced
+- as: aws-rosa-hcp-upgrade-with-running-cluster-f999
+  cron: 0 2 20 5 *
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      E2E_RUN_TAGS: '@rosa'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;ROSA&
```

Comment:
There should be an env named NAME_PREFIX being exposed. It can has a default value, but should can be configured in job level

Reply:
Updated